pandas/_libs/tslibs/period.pyx

```diff
@@ -1209,8 +1209,7 @@ def period_format(int64_t value, int freq, object fmt=None):
         elif freq_group == 4000:  # WK
             left = period_asfreq(value, freq, 6000, 0)
             right = period_asfreq(value, freq, 6000, 1)
-            return '%s/%s' % (period_format(left, 6000),
-                              period_format(right, 6000))
+            return f"(period_format(left, 6000)/period_format(right, 6000))"
```

Comment:
```suggestion
            return f"{period_format(left, 6000}/{period_format(right, 6000}"
```

Reply:
just pushed this change.  it fixed the errors for locally